src/hardware/voodoo.cpp

```diff
@@ -937,7 +938,9 @@ struct triangle_worker
 	poly_vertex v1, v2, v3;
 	INT32 v1y, v3y, totalpix;
 	SDL_sem* sembegin[TRIANGLE_THREADS];
-	std::atomic_bool done[TRIANGLE_THREADS];
+	std::condition_variable done_cv;
+	std::mutex done_mutex;
+	int done_count;
```

Comment:
Suggest defaults for the struct members (might as well do them all).

Reply:
does the

```cpp
	triangle_worker tworker = {};
```

in `voodoo_state` handle that? And if not, should I then init all the members in triangle_worker, in addition to my new ones?

Reply:
hmm.. I guess this starts to ripple further into more types, like `poly_vertex`.  

If it's passing CI then at least `-Weffc++` is satisfied there aren't any undefined states, so might as well defer my comment for a focused cleanup PR.